src/ui/CountryFlagIcon/flags/us.svg

```diff
@@ -0,0 +1,93 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" viewBox="0 0 473.677 473.677" style="enable-background:new 0 0 473.677 473.677;" xml:space="preserve">
```

Comment:
Why not something from `iconify`?

Reply:
None of the iconfiy flag icon looks good... and also, we want square-shaped icon without having to distort the image.

Reply:
Isn't it a round icon?

Reply:
Yea I meant round. I said square because initially I used squared ones and put borderRadius on it. But still, Iconify flags are ugly

Reply:
Isn't this one identical?

![image](https://user-images.githubusercontent.com/52666982/145924553-d1727480-3c21-4469-affb-65db6fac19ef.png)

Reply:
Didn't see that one before. I used the `CoreUI Flag` and it was terrible.

Let me change to this one

Reply:
Done